pkg/codegen/python/gen.go

```diff
@@ -559,7 +559,7 @@ async def get_package():
 					_package_ref = registerPackageResponse.ref
 	# TODO: This check is only needed for parameterized providers, normal providers can return None for get_package when we start
 	# using package with them.
-	if _package_ref is None:
+	if _package_ref is None or _package_ref is ...:
```

Comment:
Side question - this is the first time I've seen `something is ...` in python - does `...` here basically mean `a value that is not set, but is not set in a different way than None is`?

Reply:
Ah, yes, I guess I didn't know the word `sentinel` existed to describe the functional meaning that I was alluding to